src/main/scala/pipelines/images/cifar/RandomPatchCifarAugmented.scala

```diff
@@ -0,0 +1,158 @@
+package pipelines.images.cifar
+
+import scala.reflect.ClassTag
+
+import breeze.linalg._
+import breeze.numerics._
+
+import org.apache.spark.{SparkConf, SparkContext}
+import org.apache.spark.rdd.RDD
+
+import scopt.OptionParser
+
+import evaluation.{AugmentedExamplesEvaluator, MulticlassClassifierEvaluator}
+import loaders.CifarLoader
+
+import nodes.images._
+import nodes.learning.{BlockLeastSquaresEstimator, ZCAWhitener, ZCAWhitenerEstimator}
+import nodes.stats.{StandardScaler, Sampler}
+import nodes.util.{Cacher, ClassLabelIndicatorsFromIntLabels, MaxClassifier}
+
+import pipelines.FunctionNode
+import pipelines.Logging
+import utils.{MatrixUtils, Stats, Image, ImageUtils}
+
+object RandomPatchCifarAugmented extends Serializable with Logging {
+  val appName = "RandomPatchCifarAugmented"
+
+  class LabelAugmenter[T: ClassTag](mult: Int) extends FunctionNode[RDD[T], RDD[T]] {
+    def apply(in: RDD[T]) = in.flatMap(x => Seq.fill(mult)(x))
+  }
+
+  def run(sc: SparkContext, conf: RandomCifarFeaturizerConfig) {
+    // Set up some constants.
+    val numClasses = 10
+    val numChannels = 3
+    val whitenerSize = 100000
+    val augmentPatchSize = 24
+    val flipChance = 0.5
+
+    // Load up training data, and optionally sample.
+    val trainData = conf.sampleFrac match {
+      case Some(f) => CifarLoader(sc, conf.trainLocation).sample(false, f).cache
+      case None => CifarLoader(sc, conf.trainLocation).cache
+    }
+    val trainImages = ImageExtractor(trainData)
+
+    val patchExtractor = new Windower(conf.patchSteps, conf.patchSize)
+      .andThen(ImageVectorizer.apply)
```

Comment:
style nit - can we move the `andThen` to the end of the previous line and remove the spurious `.`, `(`, `)` and `apply` here? I know you're copy/pasting old code, but I like that style better.